src/test/ui/resolve/issue-21221-1.stderr

```diff
@@ -16,10 +18,11 @@ error[E0412]: cannot find type `Mul` in this scope
    |                ^^^ not found in this scope
    |
 help: possible candidates are found in other modules, you can import them into scope
-   | use mul1::Mul;
-   | use mul2::Mul;
-   | use mul3::Mul;
-   | use mul4::Mul;
+   |
+11 | use mul1::Mul;
+11 | use mul2::Mul;
+11 | use mul3::Mul;
+11 | use mul4::Mul;
```

Comment:
I think it might take a second look to recognize the line number is always the same. I'm not sure what scenario you want to optimize for. Many suggestions will probably look like your screenshot.

- - -

What do you think about something like:

```
help: possible candidates are found in other modules, you can import them into scope
   |
1) | use mul1::Mul;
2) | use mul2::Mul;
3) | use mul3::Mul;
4) | use mul4::Mul;
   |
```

Actually, let's think of a more complex example to see where this layout breaks: How does a diagnostic message look that

- has multiple alternative,
- that span multiple lines,
- and each has a highlight/underline?